lib/transformations/integration.test.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+const transform = require('./').transform;
+const no_of_files = 1;
+
+const runTest = (currentConfigPath) => {
+
+	let currentConfig = fs.readFileSync(currentConfigPath, 'utf8');
+	const outputConfig = transform(currentConfig);
+	expect(outputConfig).toMatchSnapshot();
+};
+
+describe('Run integration tests', () => {
+	for(let i =1; i<= no_of_files; i++){
```

Comment:
Not sure how to pick up the test files. I took a very naive approach. Should I use regex for this then?